tests/test_models/test_voxel_encoder/test_voxelize.py

```diff
@@ -14,6 +14,8 @@ def _get_voxel_points_indices(points, coors, voxel):
 
 
 def test_voxelization():
+    if not torch.cuda.is_available():
+        pytest.skip('test requires GPU and torch+cuda')
```

Comment:
Can we run this test with only cpu before? Does it use CPU-version voxelization?

Reply:
Sure, we can use the CPU-version voxelization. But the CPU-version voxelization in mmcv has something wrong. The fixing processing of CPU-version voxelization is still in process. More details can be seen [here](https://github.com/open-mmlab/mmcv/pull/1746).

Reply:
If we use the CPU-version voxelization now, this PR cannot pass the unittest. Or we can wait for the merging of the PR of fixing the CPU-version voxelization.

Reply:
OK, we can temporarily use this judgment for now while also adding unit test for CPU-version afterward.